### ISSUE TITLE
Create the home page

### ISSUE THREAD

**Issue by sibasmarak** [sibasmarak/writeitout.github.io#1]
Create a cool first page for login and search.

**Comment by susanka068** [comment]
I'd like to work on this issue !

**Comment by sibasmarak** [comment]
Yeah! Join the slack channel. You should be having experience in Web Devt.
, I suppose? Like CSS3, JavaScript and Responsive Design?


*Thanks and Regards*
*Siba Smarak Panigrahi*
*Second-Year Undergraduate*
*Department of Computer Science and Engineering*
*Indian Institute of Technology, Kharagpur*
*sibasmarak.p@iitkgp.ac.in <sibasmarak.p@iitkgp.ac.in> *
*sibasmarak.p@gmail.com <sibasmarak.p@gmail.com>*

*Contact No- (+91) 943-944-0550  |  (+91) 977-636-2162*
*LinkedIn <https://www.linkedin.com/in/siba-smarak/> | GitHub
<https://github.com/sibasmarak>*

[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
12/10/19,
07:25:10 AM

**Comment by susanka068** [comment]
I've added CSS in the homepage ! Here is the screenshot 
![Screenshot_20191211_072941](https://user-images.githubusercontent.com/40341077/70567533-206c6e00-1bbc-11ea-88a0-b727471b64f4.png) ! Please Merge the PR

**Comment by susanka068** [comment]
Is there any issue ! you're not accepting the PR

**Comment by sibasmarak** [comment]
Busy a bit! Tonight, I would check it sorry for taking time

**Comment by susanka068** [comment]
Okey , Do let me know if you have any specific requirements

**Comment by sibasmarak** [comment]
Yeah, it looks better now! But that was for the first page only, not for the associated poems or stories or others. And I actually want to sort of say that this is, let say my page where anyone can check literary work, but I want this to be for a lot of students. So I require Database Management, but we will see that later. 
Coming to the first page, It should be like a login page, to enter details and get into your page/account. You can keep a search bar. If you search for someone there, you can get to their page.
Btw nice work!
